src/App.tsx

```diff
@@ -18,7 +18,7 @@ import { theme } from './theme';
 import { EditorThemeProvider } from './themeContext';
 import { EmbedContext, EmbedMode } from './types';
 import { useInterpretCanvas } from './useInterpretCanvas';
-import { useRouter } from 'next/router';
+import router, { useRouter } from 'next/router';
```

Comment:
Is this import used anywhere?

Reply:
Yes. https://github.com/statelyai/xstate-viz/pull/255/files/d688d5f429313320ede52ce00c7bf105066d073c#diff-26ad4b834941d9b19ebf9db8082bd202aaf72ea0ddea85f5a8a0cb3c729cc6f2R101

Reply:
Shouldn't this come from the `useRouter` itself? Not sure what using this singleton gives us in terms of benefit

Reply:
oh right. yeah this can come from the named exports too.